boards/arm/96b_carbon_nrf51/doc/96b_carbon_nrf51.rst

```diff
@@ -117,6 +119,31 @@ output to a currently unused pin connected to the STM32F401RET. Users
 who are experienced in electronics rework can remove a resistor (R22)
 on the board and attach a wire to the nRF51822's UART output.
 
+.. _96b_carbon_nrf51_bluetooth:
+
+Providing Bluetooth to 96b_carbon
+*********************************
+
+This 96b_carbon_nrf51 Zephyr configuration can be used to provide
+Bluetooth functionality from the secondary nRF51832 chip to the
+primary STM32F401RE chip on the :ref:`96b_carbon_board`.
+
+To do this, build the ``samples/bluetooth/hci_spi/`` application
+provided with Zephyr with ``BOARD=96b_carbon_nrf51``, then flash it to
+the nRF51832 chip using the instructions :ref:`above
+<96b_carbon_nrf51_programming>`. (For instructions on how to build a
+Zephyr application, see :ref:`build_an_application`.)
+
+.. warning::
+
+   **Do not flash the hci_spi application on the STM32F401RET chip on
```

Comment:
Instead of telling them what not to do, tell them what to do.  Replace the warning text with:

     Be sure to flash the hci_spi application to the nRF51832 chip and not to  
     the main STM32F401RET chip.  While both chips are supported by Zephyr,
     the hci_spi application providing Bluetooth support, will only run on
     the nRF51832 chip.